doc/releasenotes.html

```diff
@@ -59,6 +59,88 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
 
 -->
 
+<h1 id="2021-12-02-r2">Version 2021.12.02 Revision 2: June 6 , 2022</h1>
+
+<p>This is a maintenance update to <a href="#2021-12-02">SAM 2021.12.02 Revision 1</a>.</p>
+
+<ul>
+<li>Improve feeback for parametric simulation graphs when they cannot be displayed. (<a href="https://github.com/NREL/SAM/pull/1051">1051</a>)</li>
```

Comment:
Spelling error in "feedback"

Reply:
Thanks! Fixed in https://github.com/NREL/SAM/pull/1070/commits/fb1cb15b7c84f98458336dc53c0e3f0fa1d614f3.